pages/en/lb4/Schemas.md

```diff
@@ -144,6 +144,85 @@ class Customer extends Entity {
 }
 ```
 
+### JSON Schema inference
+With type information inferred from using `@model` and `@property` decorators,
+we've created a module to easily build a JSON Schema from a decorated model.
```

Comment:
Delete the first sentence:
+With type information inferred from using `@model` and `@property` decorators,
 +we've created a module to easily build a JSON Schema from a decorated model.